bridges/evm/contracts/BandChainLib.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity 0.5.14;
+
+library BandChainLib {
+  function toUint64List(bytes memory _data) internal pure returns(uint64[] memory) {
+    uint64[] memory result = new uint64[](_data.length / 8);
+    require(result.length * 8 == _data.length, 'DATA_LENGTH_IS_INVALID');
+    
+    for (uint256 i = 0; i < result.length; i++) {
+      bytes8 tmp;
```

Comment:
let's make this `uint64` and update line 11 appropriately.

Reply:
done